tests/index.rs

```diff
@@ -1,4 +1,4 @@
-use super::*;
+use {super::*, crate::command_builder::ToArgs};
```

Comment:
What is the import used for? Can we remove it?

Reply:
Yes, we need it for the `to_args()` function call in one of the tests.